docs/root/configuration/listeners/network_filters/_include/wasm_filters.yaml

```diff
@@ -0,0 +1,37 @@
+static_resources:
+  listeners:
+  - address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 8000
+    filter_chains:
+    - filters:
+        - name: envoy.filters.network.wasm
+          typed_config:
+            "@type": type.googleapis.com/envoy.extensions.filters.network.wasm.v3.Wasm
+            config:
+              name: "my_plugin"
+              vm_config:
+                runtime: "envoy.wasm.runtime.v8"
+                code:
+                  local:
+                    filename: "../../../../../../../examples/wasm-cc/lib/envoy_filter_http_wasm_example.wasm"
```

Comment:
by my calculation the relative path here is `"../../../../examples/...`

but im not sure if that is the error in ci

ill try testing this config when i get a chance

also im wondering whether it would be easier (and possible) to just link to the existing yaml file rather than linking to the wasm bin in a new one

Reply:
thinking on this further

this wont work

the path that needs to be correct is in the config_validation tests

that **must** be `lib/x/y/z`

therefore it would be a lot easier just pointing the literalinclude to the existing yaml example file (as done elsewhere i think for this reason)

Reply:
Unfortunately, there is no example yaml containing `envoy.filters.network.wasm`.
It seems a lot work to create an example for `envoy.filters.network.wasm`.
Is it ok to simply rewrite `wasm_filter.rst` and not including?